documentation/handling_participants_moving_from_a_frozen_cohort.md

```diff
@@ -0,0 +1,32 @@
+# Handling participants moving from a cohort frozen for payments to the active cohort
+
+To enable the 2021 cohort to be frozen for payments, participants that had not completed their induction could be moved to the 2024 cohort by following these steps in order:
+
+1. A change happens to the participant such as changing from `withdrawn` to `active` status, transferring to another school or a change of mentor, that indicates they are continuing with their training
+2. We verify that the participant is eligible to move
+3. We move the participant
```

Comment:
For school transfers, mentor-ECT associations, revalidations,...  the code will move the participant automatically to the active cohort from the frozen one if they are eligible.